src/cloud/social/provider.ts

```diff
@@ -58,6 +58,14 @@ interface SavedContact {
   description?: string;
 }
 
+var UserStatus = {
```

Comment:
Ah, great :-) In that case, how about:
 * add a comment explaining what this duplicates
 * making this an enum too
 * make the value names match

Separately, what do you think of moving the cloud social provider to the uproxy repo? Seems like this sort of thing will happen a lot.